controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiTest.java

```diff
@@ -1644,13 +1662,19 @@ private RequestBuilder data(HttpEntity data) {
         private RequestBuilder oktaAccessToken(OktaAccessToken oktaAccessToken) { this.oktaAccessToken = oktaAccessToken; return this; }
         private RequestBuilder contentType(String contentType) { this.contentType = contentType; return this; }
         private RequestBuilder recursive(String recursive) { this.recursive = recursive; return this; }
+        private RequestBuilder header(String name, String value) {
+            this.headers.putIfAbsent(name, new ArrayList<>());
```

Comment:
`this.headers.putIfAbsent(name, new ArrayList<>()).add(value);`

Reply:
Doesn't allow adding multiple values for the same header.

Reply:
It should have identical semantics as the two code lines you wrote.

Reply:
Oh, sorry. Yes, it would have, if not for the return value being `null` when the key was previously unmapped.

Reply:
Good point - `putIfAbsent` did not return the newly inserted value as I assumed